### PR TITLE
xsv: add page

### PR DESCRIPTION
**Pull request by jedahan**
- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

---

A first crack at #2333

- Should this use `{{path/to/worldcities.csv}}` instead of `{{worldcities.csv}}`?
- Should I describe that worldcities.csv can be downloaded with `curl -LO http://burntsushi.net/stuff/worldcitiespop.csv`?

### CODE REVIEW

**Comment by mfrw** [comment]
> * Should this use `{{path/to/worldcities.csv}}` instead of `{{worldcities.csv}}`?

`{{wordcities.csv}}` should suffice.
> * Should I describe that worldcities.csv can be downloaded with `curl -LO http://burntsushi.net/stuff/worldcitiespop.csv`?

I don't think we generally provide this information but in this case, we could, for coherency.
\cc @agnivade @sbrl @pxgamer

**Comment by sbrl** [comment]
> * Should this use `{{path/to/worldcities.csv}}` instead of `{{worldcities.csv}}`?

We do prefer `{{path/to/worldcities.csv}}`, if I recall correctly, yeah.

File: pages/common/xsv.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- Get an overview of the shape of entries:
+
+`xsv stats {{worldcities.csv}} | xsv table`
+
+- Select a few columns:
+
+`xsv select {{Country,AccentCity}} {{worldcities.csv}}`
+
+- Show 10 random sample of entries:
```
Comment (sbrl) [inline pages/common/xsv.md]: Perhaps `Show a 10 random entries:` would read better:

File: pages/common/xsv.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# xsv
+
+> A fast CSV command line toolkit written in Rust.
+
+- Inspect the headers of a file:
+
+`xsv headers {{path/to/worldcities.csv}}`
```
Comment (agnivade) [inline pages/common/xsv.md]: worldcities sounds odd. We generally use path/to/file.csv. Can we use that everywhere ?

File: pages/common/xsv.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`xsv count {{path/to/worldcities.csv}}`
+
+- Get an overview of the shape of entries:
+
+`xsv stats {{path/to/worldcities.csv}} | xsv table`
+
+- Select a few columns:
+
+`xsv select {{Country,AccentCity}} {{path/to/worldcities.csv}}`
```
Comment (agnivade) [inline pages/common/xsv.md]: You can change it to column_1, column_2

File: pages/common/xsv.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+`xsv select {{column_a,column_b}} {{path/to/file.csv}}`
+
+- Show 10 random entries:
+
+`xsv sample {{10}} {{path/to/file.csv}}`
+
+- Join a column from one file to another:
+
+`xsv join --no-case {{key_a}} {{path/to/file/a.csv}} {{column_b}} {{path/to/file/b.csv}} | xsv table`
```
Comment (agnivade) [inline pages/common/xsv.md]: Question: Why is the first one called key_ and the second one column_ ? Shouldn't both be "column_x"

Comment (jedahan) [reply pages/common/xsv.md]: I was trying to hint at what join does but maybe its more confusing than not? will switch

Comment (agnivade) [reply pages/common/xsv.md]: If it is like a sql-join where values of one column should be equal to values of another column, then yes I think column_a and column_b sounds better. I was curious to know why you mentioned "key".

**Comment by sbrl** [comment]
Thanks for sticking with the review process, @jedahan :smiley_cat:

**Pull request merged**